.github/workflows/run_periodic_tests.yml

```diff
@@ -124,3 +124,7 @@ jobs:
       - name: Install dev dependencies and run example tests
         if: matrix.os == 'ubuntu-latest'
         run: tox -e examples
+
+      - name: Install scikits.odes and test pybamm_install_odes
+        if: matrix.os == 'ubuntu-latest'
+        run: python pybamm/install_odes.py
```

Comment:
Why was the `tox` installation reverted?

Reply:
I was just checking if it could install odes here too as it worked on my local system but it didn't, so I reverted back to tox.